.golangci.yaml

```diff
@@ -0,0 +1,34 @@
+run:
+  # timeout for analysis, e.g. 30s, 5m, default is 1m
+  deadline: 90s
+
+linters-settings:
+  gocyclo:
+    # minimal code complexity to report, 30 by default (but we recommend 10-20)
+    min-complexity: 25
+  depguard:
+    list-type: blacklist
+    packages:
+      - golang.org/x/net/context
+      - github.com/gogo/protobuf/proto
+
+linters:
+  disable-all: true
+  enable:
+    - deadcode
+    - depguard
+    - gocyclo
+    - gofmt
+    - goimports
+    - govet
+    - ineffassign
+    - megacheck
+    - misspell
+    - revive
+    - varcheck
+    # TODO(gbelvin): write license linter and commit to upstream.
+    # ./scripts/check_license.sh is run by ./scripts/presubmit.sh
```

Comment:
Is this still relevant?

Reply:
I was going to add these scripts as the next PR, but you're right. Either way, Gary ain't going to be fixing anything on this I doubt :-)